egs/fisher_swbd/s5/local/fisher_swbd_prepare_dict.sh

```diff
@@ -112,8 +112,12 @@ srcdict=$srcdir/swb_ms98_transcriptions/sw-ms98-dict.text
 # assume swbd_p1_data_prep.sh was done already.
 [ ! -f "$srcdict" ] && echo "No such file $srcdict" && exit 1;
 
+rm -f $dir/lexicon0.txt
 cp $srcdict $dir/lexicon0.txt || exit 1;
-patch <local/dict.patch $dir/lexicon0.txt || exit 1;
+chmod +x $srcdict $dir/lexicon0.txt
```

Comment:
This doesn't make sense to me.  why make them executable?

Reply:
sorry I meant +w. will fix